sycl/include/syclcompat/launch_policy.hpp

```diff
@@ -0,0 +1,282 @@
+/***************************************************************************
+ *
+ *  Copyright (C) Codeplay Software Ltd.
+ *
+ *  Part of the LLVM Project, under the Apache License v2.0 with LLVM
+ *  Exceptions. See https://llvm.org/LICENSE.txt for license information.
+ *  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  SYCL compatibility extension
+ *
+ *  launch.hpp
+ *
+ *  Description:
+ *    launch functionality for the SYCL compatibility extension
+ **************************************************************************/
+
+#pragma once
+
+#include "sycl/ext/oneapi/experimental/enqueue_functions.hpp"
+#include "sycl/ext/oneapi/properties/properties.hpp"
+#include <sycl/event.hpp>
+#include <sycl/nd_range.hpp>
+#include <sycl/queue.hpp>
+#include <sycl/range.hpp>
+
+#include <syclcompat/defs.hpp>
+#include <syclcompat/device.hpp>
+#include <syclcompat/dims.hpp>
+#include <syclcompat/traits.hpp>
+
+namespace syclcompat {
+namespace experimental {
```

Comment:
Are we gonna release this under the experimental namespace?

Reply:
Yes, I think so